src/session/content/message_row/document/status_indicator.rs

```diff
@@ -56,6 +56,10 @@ mod imp {
         fn property(&self, id: usize, pspec: &glib::ParamSpec) -> glib::Value {
             self.derived_property(id, pspec)
         }
+
+        fn dispose(&self) {
+            self.status_image.unparent();
```

Comment:
You can use `dispose_template` nowadays.

Reply:
> You can use `dispose_template` nowadays.

TIL. Do you know whether there's a reason why unparenting isn't handled automatically?

Reply:
> TIL

Note that this will only unparent the widgets that are bound via the template_child macro.

> Do you know whether there's a reason why unparenting isn't handled automatically?

Apart of the current restrain on API compatibility, I'm not sure. I would like automatic unparenting too with gtk5 honestly.

Reply:
> Note that this will only unparent the widgets that are bound via the template_child macro

I just noticed this :cry:.